protoc-gen-go/grpc/grpc.go

```diff
@@ -117,8 +117,7 @@ func (g *grpc) Generate(file *generator.FileDescriptor) {
 	g.P("// Reference imports to suppress errors if they are not otherwise used.")
 	g.P("var _ ", contextPkg, ".Context")
 	g.P("var _ ", grpcPkg, ".ClientConn")
-	g.P("var errUnimplemented = func (methodName string) error {\n\t return ", errorPkg, ".Errorf(codes.Unimplemented, \"Method %s not implemented\", methodName)\n}")
-	g.P("var _ ", codePkg, ".Code")
+	g.P("var errUnimplemented = func(methodName string) error {\n\t return ", errorPkg, ".Errorf(codes.Unimplemented, \"method %s not implemented\", methodName)\n}")
```

Comment:
I think there is still my specific point that you need `\n\treturn` not `\n\t return`.

Reply:
Fixed.